### PR TITLE
Bump(pyavd): Move PyAVD out of beta

### PR DESCRIPTION
**Pull request by ClausHolbechArista**
Release PyAVD (remove beta)

### CODE REVIEW

File: ansible_collections/arista/avd/docs/pyavd.md [hunk 1]
```diff
@@ -31,7 +28,7 @@ Feedback is very welcome. Please use [GitHub discussions](https://github.com/ari
 ## Known limitations
 
 - No support for inline Jinja2 or custom Jinja2 templates.
-- Input data and "structured_configs" will be in-place updated by various pyavd functions.
+- Input data and "structured_configs" will be in-place updated by various PyAVD functions.
   Make sure to deep copy the data first if modifications are not allowed.
```
Comment (gmuloc) [inline ansible_collections/arista/avd/docs/pyavd.md]: I would make this a bit more "visible" maybe a note or .. I don't know to draw attention

File: ansible_collections/arista/avd/docs/pyavd.md [hunk 2]
```diff
@@ -47,13 +44,13 @@ Feedback is very welcome. Please use [GitHub discussions](https://github.com/ari
...
-- Add cloudvision tag integrations
-- Make pyavd the source of AVD logic and use as a dependency for the `arista.avd` Ansible collection.
+- Add CloudVision tag integrations
+- Make PyAVD the source of AVD logic and use as a dependency for the `arista.avd` Ansible collection.
 - Explore support for custom Jinja2 templates.
 
 ## Installation
 
-Install `pyavd`:
+Install the `pyavd` Python package:
```
Comment (gmuloc) [inline ansible_collections/arista/avd/docs/pyavd.md:61]: do we want to maintain the requirements versions in our doc going forward?

**Review by gmuloc — COMMENTED** [review]
I think you should also change the classifier in pyproject.toml

https://github.com/aristanetworks/ansible-avd/blob/devel/python-avd/pyproject.toml#L10

File: ansible_collections/arista/avd/docs/pyavd.md (hunk 2 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/docs/pyavd.md:61]: There is no alternative right now (I tried!). The mkdocs root is the ansible collection, so we cannot read a snip from outside.

Comment (gmuloc) [reply ansible_collections/arista/avd/docs/pyavd.md:61]: ack - we can deal with this later once we move the doc at the top level of the repo

**Review by gmuloc — APPROVED** [review]
LGTM ✅

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Pull request merged**